test/examples/secure_threat_detection/single/main.tf

```diff
@@ -0,0 +1,20 @@
+provider "aws" {
+  region      = "us-east-1"
+  access_key  = "test"
+  secret_key  = "test"
+
+  endpoints {
+    iam     = "http://127.0.0.1:5000/"
```

Comment:
These should be externalized and shared among different test examples.

Reply:
Are you planning to have all combinations of provider's and modules here? 
I assume this is what the user gets as the return for installActions api?

Reply:
> Are you planning to have all combinations of provider's and modules here?

if externalized, yes. if not, no

> I assume this is what the user gets as the return for installActions api?

not this particular line, no, but the contents of this file is intended to generally resemble a working install.